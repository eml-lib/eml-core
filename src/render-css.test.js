import renderCss from './render-css';

it('Smthng', () => {
	expect(
		renderCss(
			{
				'table': {
					border: '1px solid red'
				},
				'@media (max-width: 100px)': {
					'.a, .b, .c': {
						width: 100,
						height: 200
					},
					'.rule2': {
						height: '100px',
						backgroundColor: 'red',
						WebkitFlex: 1
					}
				}
			}
		)
	).toMatchSnapshot();
});