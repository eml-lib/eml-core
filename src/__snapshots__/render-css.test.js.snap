// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Smthng 1`] = `
"table {
	border: 1px solid red
}

@media (max-width: 100px) {
	.a, .b, .c {
		width: 100px; height: 200px
	}
	.rule2 {
		height: 100px; background-color: red; -webkit-flex: 1
	}
}
"
`;
